docs/UserGuide.md

```diff
@@ -172,6 +172,20 @@ Examples:
 * `list` followed by `listt 1` lists transactions for the 1st person in the address book.
 * `find Betsy` followed by `listt 1` lists the transactions for the 1st person in the results of the `find` command.
 
+### Deleting Transactions for a specified person: `deletet`
+
+Format: `deletet INDEX`
+
+* Deletes the transactions at the specified `INDEX` for the first person on the current list.
```

Comment:
Do you want to rephrase it like "Deletes the transaction at the specified 'INDEX' in the current transaction  list"? I feel like users might be confused by "the first person".

Reply:
Ok, that sounds better, didn't know how to phrase it.